microprofile-metrics/pom.xml

```diff
@@ -37,5 +37,34 @@
             <artifactId>wildfly-arquillian-container-managed</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
```

Comment:
put it in dependency management

why is it needed ? jax-rs is part of mp spec and jax-rs is based on servelt

Reply:
Fixed.